src/PrestaShopBundle/Resources/views/Admin/Product/Include/form_combination.html.twig

```diff
@@ -1,3 +1,4 @@
+{% set is_stock_management = 'PS_STOCK_MANAGEMENT'|configuration %}
```

Comment:
Since this is used in multiple twig files, and it rely on global store configuration, you could move this to twig global assignation. I discussed this with @thierrymarianne, you'll have to add something like this to `app/config/config.yml`

```yml
twig:
    globals:
        user_management: "@=service('prestashop.adapter.legacy.configuration').get('PS_STOCK_MANAGEMENT')"
```

then you can remove this line (and all similar code).